compiler/one-cmds/one-optimize

```diff
@@ -51,6 +51,14 @@ def _get_parser():
         action='store_true',
         help="""Experimental: This will convert NCHW operators to NHWC under the
         assumption that input model is NCHW.""")
+    circle2circle_group.add_argument(
+        '--nchw_to_nhwc_preserve_input_shape',
+        action='store_true',
+        help='preserve the input shape of the model')
```

Comment:
Maybe would be better to add comments as `circle2circle` comments.

Reply:
Updated. Thanks :)